document_page_approval/views/document_page_approval.xml

```diff
@@ -164,7 +164,7 @@
                 <field name="has_changes_pending_approval" invisible="1" />
                 <field name="user_has_drafts" invisible="1" />
             </sheet>
-            <button name="toggle_active" position="after">
+            <widget name="web_ribbon" position="after">
```

Comment:
Why on web ribbon? Can you use the button_box?

Reply:
I thought that [here](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0) it was commented that the new paradigm is to use the web_ribbon

Reply:
web_ribbon is used to show that the element is archived, toggle_button needs to be used for buttons, so change 
`<widget name="web_ribbon" position="after">`
for
`<xpath expr="//div[@name='button_box']" position="inside">`

Reply:
Oh! Sorry! I understand. They are issues that have nothing to do with each other. Change made.